### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by svyshcho**

### CODE REVIEW

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+import sys
+
+from datetime import datetime
+
+
+def create_file() -> None:
```
Comment (EdAlekseiev) [inline app/create_file.py:7]: Please, separate this function into a few small functions.

Comment (svyshcho) [reply app/create_file.py]: Done

File: app/create_file.py [hunk 2]
```diff
@@ -1 +1,39 @@
...
+        os.makedirs(path, exist_ok=True)
+        file_path = os.path.join(path, command[-1])
+        open_file(file_path)
+
+    if command[1] == "-d" and "-f" not in command:
+        os.makedirs(os.path.join(*command[2:]))
+
+
+def open_file(command: str) -> None:
+    with open(command, "w") as file:
```
Comment (EdAlekseiev) [inline app/create_file.py:26]: If the file exists app should add content below. App rewrites file content now.

File: app/create_file.py [hunk 3]
```diff
@@ -1 +1,39 @@
...
+        open_file(command[-1])
+
+    if command[1] == "-d" and "-f" in command:
+        path = os.path.join(*command[2:-2])
+        os.makedirs(path, exist_ok=True)
+        file_path = os.path.join(path, command[-1])
+        open_file(file_path)
+
+    if command[1] == "-d" and "-f" not in command:
+        os.makedirs(os.path.join(*command[2:]))
```
Comment (EdAlekseiev) [inline app/create_file.py:22]: It does not work for `python create_file.py -f file.txt -d dir1 dir2`

File: app/create_file.py [hunk 4]
```diff
@@ -1 +1,39 @@
...
+import os
+import sys
+
+from datetime import datetime
+
+
+command = sys.argv
+
+
+def create_file(command: list) -> None:
```
Comment (EdAlekseiev) [inline app/create_file.py:10]: What happens if the command will be empty?

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — APPROVED** [review]
Well done!

**Pull request closed without merging**